### ISSUE TITLE
unable to curl request airflow with 401

### ISSUE THREAD

**Issue by Sms-Rk** [bitnami/containers#51069]
### Name and Version

bitnami/airflow:2.7.1

### What architecture are you using?

None

### What steps will reproduce the bug?

in docker swarm environment with this config

```yaml
"environment": [
    "AIRFLOW_DATABASE_NAME: airflow",
    "AIRFLOW_DATABASE_USERNAME: postgres",
    "AIRFLOW_DATABASE_PASSWORD: ***",
    "AIRFLOW_EXECUTOR: CeleryExecutor",
    "AIRFLOW_DATABASE_HOST: airflow-db",
    "AIRFLOW_USERNAME: ***",
    "AIRFLOW_PASSWORD: ***"
],
"volumes": [
    "/docker/data/airflow/dags:/opt/bitnami/airflow/dags"
```
I have a worker and a scheduler that didn't put config here and edit _airflow.cfg_ inside the container to this
```
[webserver]
base_url=http://127.0.0.1:8080
rbac=true
web_server_port=8080

[core]
load_examples=True
executor=CeleryExecutor

[database]
sql_alchemy_conn=postgresql+psycopg2://postgres:***@airflow-db:5432/airflow

[celery]
broker_url=redis://:@redis:6379/1
result_backend=db+postgresql://postgres:***@airflow-db:5432/airflow

[api]
auth_backends=airflow.api.auth.backend.basic_auth
```
behind the nginx with using this curl command
```
curl -u '***:***' -H 'Accept: application/json' -X GET 'https://***/api/v1/dags'
```
I get 
```
{
  "detail": null,
  "status": 401,
  "title": "Unauthorized",
  "type": "https://airflow.apache.org/docs/apache-airflow/2.7.1/stable-rest-api-ref.html#section/Errors/Unauthenticated"
}
```

### What do you see instead?
```
{
  "detail": null,
  "status": 401,
  "title": "Unauthorized",
  "type": "https://airflow.apache.org/docs/apache-airflow/2.7.1/stable-rest-api-ref.html#section/Errors/Unauthenticated"
}
```

**Comment by carrodher** [comment]
It appears that the issue may not be directly related to the Bitnami Airflow container image or Helm chart, but rather to how the application is being utilized or configured in your specific environment.

For inquiries about the application itself, customizing its content, or questions about technology and infrastructure usage, we highly recommend referring to the forums and user guides provided by the project responsible for the application or technology.

With that said, we'll keep this ticket open until the stale bot automatically closes it, in case someone from the community contributes valuable insights.

**Comment by baluchen1** [comment]
I have the same issue. Can you confirm if this is fixed? or what is the fix? I'm facing with Airflow 2.6
